src/Modules/Workspaces/Commands.Workspaces/GetPowerBIWorkspace.cs

```diff
@@ -154,7 +154,7 @@ private void GetWorkspaces()
 
                 var workspaces = this.Scope == PowerBIUserScope.Individual ?
                     client.Workspaces.GetWorkspaces(filter: this.Filter, top: this.First, skip: this.Skip) :
-                    client.Workspaces.GetWorkspacesAsAdmin(expand: "users", filter: this.Filter, top: this.First, skip: this.Skip);
+                    client.Workspaces.GetWorkspacesAsAdmin(top: this.First.GetValueOrDefault(), expand: "users", filter: this.Filter, skip: this.Skip);
```

Comment:
We should consider updating first to be a mandatory parameter so we don't have to do GetValueOrDefault() now that it's required by the SDK. We might be able to remove some of the logic that sets a default value since this is a new version.